src/models/meme.ts

```diff
@@ -0,0 +1,20 @@
+class MemeResource {
+  id: string;
+  title: string;
+  description: string;
+  link: string;
+  nsfw: boolean;
+  type: string;
+  created_at: Date;
```

Comment:
Not needed in constructor because of `table.dateTime("created_at").defaultTo(knex.fn.now());` in `create-meme-table.ts`?